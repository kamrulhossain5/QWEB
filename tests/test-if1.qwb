function int main(){
  if (true){
    print(19);
  }
  print(17);
}
