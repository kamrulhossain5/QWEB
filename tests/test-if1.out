19<br>
17<br>
